src/Network/Receive/ServerType0.pm

```diff
@@ -1153,14 +1162,16 @@ sub map_loaded {
 		main::initMapChangeVars();
 	} else {
 
-		$messageSender->sendSync(1) if ($masterServer->{serverType} eq 'bRO'); # tested at bRO 2013.11.26 - revok
+		$messageSender->sendMapConnected if $messageSender->can( 'sendMapConnected' );
+
+		$messageSender->sendSync(1);
 
 		$messageSender->sendGuildMasterMemberCheck();
 
 		# Replies 01B6 (Guild Info) and 014C (Guild Ally/Enemy List)
 		$messageSender->sendGuildRequestInfo(0);
 
-		$messageSender->sendGuildRequestInfo(0) if ($masterServer->{serverType} eq 'bRO'); # tested at bRO 2013.11.26, this is sent two times and i don't know why - revok
+		$messageSender->sendGuildRequestInfo(0);
```

Comment:
> Maybe instead of a check on the servertype, we should instead do a configuration option in servers.txt ? Maybe something like requestGuildInfoTwice.

Since official clients request it twice, "requestGuildInfoTwice" should maybe default to true?